maintainers/team-list.nix

```diff
@@ -857,8 +857,7 @@ with lib.maintainers; {
   python = {
     members = [
       hexa
-      jonringer
```

Comment:
I was banned for six weeks, and as active in the python-updates preceding the ban, I would like to remain

Reply:
Was hard to judge, given that you were also away for 2 years before that.

Reply:
mid 2022 to late 2023. Which is fair.

Either way, I'm happy to help maintain it.